distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/Client.java

```diff
@@ -13,121 +13,111 @@
-                }
-                LOGGER.info("-----------------------");
-
-                // OrderMgr
-                if (SHOULD_PROCESS_PKG_ELEMENTS) {
-                    PackageElement[] packageElements = nsiliClient.order(orb, results);
-
-                    // ProductMgr
-                    // For each packageElement in the order response, get the parameters and
-                    // related files for the product.
-                    if (packageElements != null) {
-                        for (PackageElement packageElement : packageElements) {
-                            Product product = packageElement.prod;
-                            nsiliClient.get_parameters(product);
-                            nsiliClient.get_related_file_types(product);
-                            nsiliClient.get_related_files(orb, product);
-                        }
+                for (int i = 0; i < results.length; i++) {
+                    LOGGER.info("\t RESULT : {} of {} ", (i + 1), results.length);
+                    if (getAttributeFromDag(results[i], NsiliConstants.STATUS).equalsIgnoreCase(
+                            NsiliCardStatus.OBSOLETE.name())) {
+                        LOGGER.info("Record is {}. Not testing result",
+                                NsiliCardStatus.OBSOLETE.name());
                     } else {
-                        LOGGER.info("Order does not have any package elements");
+                        sampleNsiliClient.printDagAttributes(results[i]);
+                        if (SHOULD_DOWNLOAD_PRODUCT) {
+                            try {
+                                sampleNsiliClient.downloadProductFromDAG(results[i]);
+                            } catch (MalformedURLException e) {
+                                LOGGER.error("Invalid URL used for product retrieval.", e);
+                            }
+                        }
+
+                        // ProductMgr
+                        LOGGER.info("-----------------------");
+                        try {
+                            sampleNsiliClient.testProductMgr(results[i]);
+                        } catch (Exception e) {
+                            LOGGER.info("Unable to test ProductMgr: {}",
+                                    NsilCorbaExceptionUtil.getExceptionDetails(e));
+                        }
+                        LOGGER.info("-----------------------");
+
+                        // OrderMgr
+                        PackageElement[] packageElements = sampleNsiliClient.order(results[i]);
+
+                        // ProductMgr
+                        if (SHOULD_PROCESS_PKG_ELEMENTS) {
+                            // TODO prod is null for packageElement https://codice.atlassian.net/browse/CAL-192
+                            for (PackageElement packageElement : packageElements) {
+                                Product product = packageElement.prod;
+                                sampleNsiliClient.get_parameters(product);
```

Comment:
The `get_parameters` prints the attributes from the `productMgr.get_parameters` response